docs/reference.html

```diff
@@ -1,4 +1,4 @@
 ---
 layout: redirected
-redirect_to: reference-1.3.0.html
+redirect_to: reference-1.3.1.html
```

Comment:
Hum maybe my previous comment was confusing…
I would keep `reference.html` to redirect directly to 1.3.0, since that is the final page, instead of going through 1.3.1 and chain 2 redirections?

Reply:
yes, it makes sense